src/components/Table.res

```diff
@@ -443,12 +443,7 @@ module TableHeadingCell = {
                     let selfClass = "self-end"
                     <div className={`flex flex-row ${selfClass} items-center`}>
                       <SortAction
-                        item
-                        sortedObj
-                        setSortedObj
-                        sortIconSize
-                        filterRow
-                        isLastCol={isLastCol && !isFrozen && !isFirstCol}
```

Comment:
Why this condition is changed ?

Reply:
Before the filter is defaultly not supported in the last colomn which shoudn't be the case